doc/plutus/tutorials/contract-models.rst

```diff
@@ -1211,8 +1211,8 @@ Here :hsobj:`Plutus.Contract.Test.ContractModel.Interface.assertModel` lets us i
 model state, :hsobj:`Plutus.Contract.Test.ContractModel.Interface.lockedValue` is a function provided by the framework
 that computes the total value held by contracts, and :hsobj:`Plutus.Contract.Test.ContractModel.Interface.symIsZero`
 checks that this is zero. The value is returned here as a
-:hsobj:`Plutus.Contract.Test.ContractModel.Symbolics.SymValue`, but for now it can be thought of just as a normal
-Plutus :hsobj:`Plutus.V1.Ledger.Value.Value` with an extra type wrapper.
+`Plutus.Contract.Test.ContractModel.Symbolics.SymValue` [TODO broken link], but for now it can be thought of just as a normal
```

Comment:
Please help: `Plutus.Contract.Test.ContractModel.Symbolics.SymValue` is obsolete. What should I replace it with?

Reply:
`Test.QuickCheck.ContractModel.Internal.Symbolics.SymValue` from the `quickcheck-contractmodel` package.

Reply:
Looks like it has been moved to https://github.com/input-output-hk/quickcheck-contractmodel/blob/master/quickcheck-contractmodel/src/Test/QuickCheck/ContractModel/Internal/Symbolics.hs